hydra-main/src/main/java/com/addthis/hydra/job/spawn/balancer/SpawnBalancer.java

```diff
@@ -1205,14 +1246,54 @@ protected void updateAggregateStatistics(Iterable<HostState> hosts) {
                 findActiveJobIDs();
                 double maxMeanActive = -1;
                 double maxDiskPercentUsed = -1;
+                long maxDiskFree = 0;
+                long minDiskFree = Long.MAX_VALUE;
+                BigInteger sumDiskFree = BigInteger.valueOf(0);
+                double maxTaskPercent = 0;
+                double minTaskPercent = 1;
+                double sumTaskPercent = 0;
                 for (HostState host : hosts) {
                     maxMeanActive = Math.max(maxMeanActive, host.getMeanActiveTasks());
                     maxDiskPercentUsed = Math.max(maxDiskPercentUsed, getUsedDiskPercent(host));
                 }
                 for (HostState host : hosts) {
-                    cachedHostScores.put(host.getHostUuid(),
-                                         calculateHostScore(host, maxMeanActive, maxDiskPercentUsed));
+                    // metrics here
+                    HostScore score = calculateHostScore(host, maxMeanActive, maxDiskPercentUsed);
+                    cachedHostScores.put(host.getHostUuid(), score);
+                    long freeDisk = score.getDiskFreeBytes();
+                    double taskPercent = score.getScoreValue(false);
+                    sumDiskFree.add(BigInteger.valueOf(freeDisk));
+                    sumTaskPercent += taskPercent;
```

Comment:
taskPercent is a value from 0 to 1, indicating how many percent of task slots have been taken up on average since the minion process started.

Currently, we only have 3 tasks per minion. That will change. So just getting the metric directly here doesn't make sense for the future.